examples/modules/modules/TCell.wyt

```diff
@@ -0,0 +1,6 @@
+type TCell
```

Comment:
This already exists in examples/modules.  Why do we need it in examples/modules/modules as well?

(same comment for other files in examples/modules/modules - best to delete them all and just use the ones in the parent directory, unless there is some reason those are unsuitable)